astrosat_users/views/views_customers.py

```diff
@@ -43,26 +45,13 @@ class CustomerViewMixin(object):
 
     # DRY way of customizing object retrieval for the 2 views below
 
+    @cached_property
+    def customer(self):
+        return self.get_object()
+
```

Comment:
Strictly speaking, factoring out this code into a separate`customer` property isn't needed since it's only used in one function: the `active_managers` property below.  But it does give me symmetry with the other Mixin below.